CHANGELOG.md

```diff
@@ -73,6 +73,9 @@ _None_
 * Strings: fix incorrect interpretation of format placeholders when there were missing positional parameters (e.g. `"%2$@"` without a `%1$…` defined).  
   [@AliSoftware](https://github.com/AliSoftware)
   [#634](https://github.com/SwiftGen/SwiftGen/pull/634)
+* Plist: fix Bool being assigned Integer when using `inline` template
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/7f25db7e56d2f6963484bdf33115e11a3eb9f3a2/CHANGELOG.md#L76"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>